.travis.yml

```diff
@@ -5,15 +5,10 @@ jdk:
 services:
   - docker
 before_install:
-  - chmod +x ./travis.sh
-  - travis_wait ./travis.sh
-before_script:
-  - |
-    statuscode=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u admin:Admin123 http://localhost:8099/openmrs/ws/rest/v1/visittype)
-    echo "Curl Status Code --> $statuscode"
-    if [ "$statuscode" -ne 200 ]; then
-      travis_terminate 1
-    fi
+  - sudo apt install jq
+  - chmod +x ./e2e-tests/batch-mode-tests.sh
+  - chmod +x ./e2e-tests/run.sh
+  - travis_wait ./e2e-tests/run.sh
 script:
-  - mvn clean install
+  - mvn clean install -DskipTests=true
```

Comment:
Why are we skipping tests?

Reply:
Was for testing purpose, I've reverted back